### PR TITLE
Added submit action for AddEmailAddressController

### PR DESCRIPTION
**Pull request by miranda-hawkes**
Also added stub action for change email for routing purposes
For review only, please don't merge until BTAT-8058 is in

### CODE REVIEW

**Comment by miranda-hawkes** [comment]
Ready for re-review @SheilaTran 😺

File: app/controllers/contactPreference/ChangeEmailController.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import controllers.predicates.AuthPredicateComponents
+import controllers.predicates.inflight.InFlightPredicateComponents
+import javax.inject.Inject
+import play.api.mvc.{Action, AnyContent}
+
+class ChangeEmailController @Inject()(implicit val appConfig: AppConfig,
+                                      authComps: AuthPredicateComponents,
+                                      inFlightPredicateComponents: InFlightPredicateComponents) extends BaseController {
+
+  //TODO: Update as part of BTAT-8061
```
Comment (SheilaTran) [inline app/controllers/contactPreference/ChangeEmailController.scala:30]: Not sure if this is still meant to be here

File: app/controllers/contactPreference/AddEmailAddressController.scala [hunk 2]
```diff
@@ -17,12 +17,14 @@
 package controllers.contactPreference
 
 import config.{AppConfig, ErrorHandler}
+import common.SessionKeys
```
Comment (SheilaTran) [inline app/controllers/contactPreference/AddEmailAddressController.scala:20]: This import isn't used anywhere, I think it's okay to delete

File: app/controllers/contactPreference/ChangeEmailController.scala (hunk 1 above)
Comment (miranda-hawkes) [reply app/controllers/contactPreference/ChangeEmailController.scala:30]: Yeah it's just a reminder as it will be completed in Deng's branch :)

**Pull request merged**